package.json

```diff
@@ -1,9 +1,10 @@
 {
   "private": true,
   "name": "karrot-frontend",
-  "productName": "Karrot",
+  "siteName": "Karrot",
```

Comment:
This should stay as `productName`, it's a field specified by quasar here https://quasar.dev/quasar-cli/quasar-conf-js#property-build

> productName	String	Default value is taken from package.json > productName field.

Reply:
Haha need to read documentation more before commits maybe :D